test/README.md

```diff
@@ -25,6 +25,10 @@ trade-off, mainly completeness vs. speed.
 #### Run the core mocha unit/integration test suite.
 `npm run test:unit`
 
+#### Run the core mocha unit/integration test suite only for a particular set of tests
+Edit the line for `test:unit:grep` on `package.json` for the pattern you want to run tests on
+Then you can run: `npm run test:unit:grep` and it will only run those tests
```

Comment:
Currently the README for `next` has no documentation for `yarn`. This PR's main purpose is to add a separate script for test:unit:grep. Not sure why you are bringing up yarn?